src/main/java/uk/gov/hmcts/reform/managecase/api/payload/ApplyNoCDecisionResponse.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.reform.managecase.api.payload;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.JsonNode;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+import java.util.Map;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@ApiModel("Verify Notice of Change Answers Response")
```

Comment:
Verify Notice of Change Answers Response --> is this correct description?

Reply:
Good spot 👍